taxonomic_dist_by_soil_layer/R/taxonomic_dist_soil_layer_R.ipynb

```diff
@@ -0,0 +1,2622 @@
+{
```

Comment:
<p>We can remove this call too and change the column<code>extraction_has_input</code> to <code>processed_sample_id2</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>